### PR TITLE
Activities query parameter

### LINKED ISSUE #122: [3] ?activity query parameter

### PR DESCRIPTION
**Pull request by tschuy**
Fixes #122

### CODE REVIEW

**Comment by LyonesGamer** [comment]
The test "returns an error for a non-existent activity" still fails.

**Comment by LyonesGamer** [comment]
It all looks good to me. I went over it a few times, but honestly I feel like I could have easily missed something. Tentative +1.

File: src/times.js [hunk 1]
```diff
@@ -8,44 +8,125 @@ module.exports = function(app) {
...
         }
+      }
 
-        for (let i = 0, len = times.length; i < len; i++) {
-          // using that user id, get the username and set it
-          // to the time user
-          times[i].user = idUserMap[times[i].user];
-        }
+      let selectedActivities = activities;
+      if (activitiesList !== undefined) {
```
Comment (Kennric) [inline src/times.js]: Weren't we just in an activitiesList !== undefined block? why another when activitiesList has not changed?

**Comment by Kennric** [comment]
This is looking pretty good. +1

**Pull request merged**